cmd/chainsimulator/main.go

```diff
@@ -334,6 +320,35 @@ func initializeLogger(ctx *cli.Context, cfg config.Config) (closing.Closer, erro
 	return fileLogging, nil
 }
 
+func fetchConfigs(skipDownload bool, cfg config.Config, nodeConfigs, proxyConfigs string) error {
+	buildInfo, ok := debug.ReadBuildInfo()
+	if !ok {
+		return errors.New("cannot read build info")
+	}
+	if skipDownload {
+		log.Warn(`flag "skip-configs-download" has been provided, if configs for node and proxy are missing simulator will not start`)
```

Comment:
if the configs*
are missing, then the simulator will not start *

Reply:
done